stackcopy.py

```
@@ -22,7 +25,98 @@
+    if IS_WINDOWS:
+        try:
+            import ctypes
+            import ctypes.wintypes
+
+            CSIDL_MYPICTURES = 0x0027
+            buf = ctypes.create_unicode_buffer(ctypes.wintypes.MAX_PATH)
+            ctypes.windll.shell32.SHGetFolderPathW(None, CSIDL_MYPICTURES, None, 0, buf)
+            if buf.value:
+                return buf.value
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)